### PR TITLE
enf-ssh: add SSH jumphost example

### PR DESCRIPTION
**Pull request by drbild**
Adds an example SSH jumphost docker image.  This was used for the Lenovo demo. I'm force-merging it now to be able to share with them in my follow-up email.  But please review when you get a chance anyway.

### CODE REVIEW

File: enf-ssh/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+EXPOSE 22
+
+VOLUME ["/data/keys"]
+
+# Rename parent entrypoint
+RUN mv /usr/local/bin/entrypoint.sh /usr/local/bin/enftun-entrypoint.sh
+
+# Configure entrypoint script
+COPY entrypoint.sh /usr/local/bin
+RUN ln -s usr/local/bin/entrypoint.sgh / # backwards compat
```
Comment (zanebeckwith) [inline enf-ssh/Dockerfile:19]: This should be `entrypoint.sh` (not `.sgh`), right?

File: enf-ssh/Makefile [hunk 2]
```diff
@@ -0,0 +1,51 @@
+VERSION := 1.0.0
+ORG := xaptum
+NAME := enf-ssh
+IMG := $(ORG)/$(NAME)
+
+## Override the on the command line
+## make USER=... ADDRESS=... DNS=...
```
Comment (zanebeckwith) [inline enf-ssh/Makefile:7]: Should the DNS setting be mentioned in the README?

**Review by zanebeckwith — COMMENTED** [review]
One possible typo, and a comment on the README.

This looks really nice, and will be super useful

File: enf-ssh/Dockerfile (hunk 1 above)
Comment (drbild) [reply enf-ssh/Dockerfile:19]: :+1:

File: enf-ssh/Makefile (hunk 2 above)
Comment (drbild) [reply enf-ssh/Makefile:7]: Yes, thanks.  I didn't want to complicate this with instructions for setting up DNS, but should mention that variable.

**Comment by drbild** [comment]
@zanebeckwith Fixed both those issues and published a new 1.0.1 release.

**Comment by zanebeckwith** [comment]
drbild go be off from work! Stop checking Github!

**Comment by drbild** [comment]
Haha, I'm floating in and out today.

**Pull request merged**